vm-setup/roles/v1aX_integration_test/tasks/move.yml

```diff
@@ -130,11 +137,26 @@
       chdir: "{{ BMOPATH }}"
 
   - name: Label BMO CRDs in target cluster.
+  - name: Restore original BMO Configmap
+    shell: "mv {{ BMOPATH }}/config/default/ironic.env.orig {{ BMOPATH }}/config/default/ironic.env"
+    when: CAPM3_VERSION != "v1alpha4"
+
+  - name: Restore original Ironic Configmap
+    shell: "mv {{ BMOPATH }}/ironic-deployment/keepalived/ironic_bmo_configmap.env.orig {{ BMOPATH }}/ironic-deployment/keepalived/ironic_bmo_configmap.env"
+
```

Comment:
Do we still have these files @lentzi90 ?

Reply:
This looks like a bad rebase. These commands were removed in https://github.com/metal3-io/metal3-dev-env/pull/975

Reply:
Fxied